documentation/tutorial/02-environment-setup.md

```diff
@@ -0,0 +1,74 @@
+# Environment Setup
+
+This is part of the
```

Comment:
This paragraph is not needed. This applies to all pages. It would be better to have a two-sentence summary of what the user is going to do in this step of the tutorial